tests/test_ingester.py

```diff
@@ -100,7 +100,16 @@ def comparator(actual, expected):
         for a, e in zip(actual, expected):
             err = abs(time.time() - a['start']/1000.0)
             assert err < 5.0
-            assert sort(a['records']) == sort(e['records'])
+            for ar, er, in zip(sorted(a['records']), sorted(e['records'])):
```

Comment:
probably want to use the `sort` function here.

Reply:
Ah yes. Fixed.